airflow/include/tasks/extract/utils/html_utils.py

```diff
@@ -23,28 +24,72 @@ def is_valid_url(url):
     return bool(parsed.netloc) and bool(parsed.scheme)
 
 
-def exclude_path(url, exclude_docs=None):
+def fetch_page_content(url: str) -> str:
+    try:
+        response = requests.get(url)
+        response.raise_for_status()  # Raise an HTTPError for bad responses
+        return response.content
+    except requests.RequestException:
+        logger.info("Error fetching content for %s: %s", url, url)
+        return ""
+
+
+def is_excluded_url(url: str, exclude_literal: list[str]) -> bool:
+    url_path = urlparse(url).path
+    return any(literal in url_path for literal in exclude_literal)
+
+
+def remove_tags(text_content: str) -> str | None:
```

Comment:
`remove_tags` is a bit confusing as we're not remove all the `tags`. I have 2 suggestions in mind. 

1. make `tags` configurable and set the default value to `script` and `style`
2. rename it as `clean_tags`, `clean_non_content_tags` (none of them is good. naming is hard 😞)

Reply:
Added tags, and changed name to clean_tags